src/utilities/apiRequests.js

```diff
@@ -8,7 +8,7 @@ export const Get = async (route) => {
     const response = await axios.get(baseUrl + route);
     return response.data;
   } catch (error) {
-    return error.response.data;
+    return error.response ? error.response.data : error;
```

Comment:
Don't we have an error handler? Or are the errors already handled somewhere?

Reply:
We are handling it in the action creators.